python/ray/rllib/common.py

```diff
@@ -30,16 +59,22 @@ class Algorithm(object):
   TODO(ekl): support checkpoint / restore of training state.
   """
 
-  def __init__(self, env_name, config):
+  def __init__(self, env_name, config, s3_bucket=None):
```

Comment:
It'd be nice to document how `s3_bucket` is being used. I was going to suggest doing that under `Attributes` above, except that it looks like it isn't actually being saved as an attribute.